boards/redboard_artemis_nano/Makefile

```diff
@@ -11,8 +11,8 @@ endif
 
 .PHONY: flash
 flash: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/release/$(PLATFORM).bin
-	python ambiq/ambiq_bin2board.py --bin $< --load-address-blob 0x20000 -b 115200 -port $(PORT) -r 2 -v --magic-num 0xCB --version 0x0 --load-address-wired 0xc000 -i 6 --options 0x1
+	ambiq/ambiq_bin2board.py --bin $< --load-address-blob 0x20000 -b 115200 -port $(PORT) -r 2 -v --magic-num 0xCB --version 0x0 --load-address-wired 0xc000 -i 6 --options 0x1
```

Comment:
Why does this help?

Does changing this to `python3` instead of `python` also work?

Reply:
It would, but the script has a shebang at the beginning, so I figure we may as well use it.

Reply:
But then you had to change it to be executable, which is a difference from the original. Not a big deal, but still something that we don't really need.